streams/kw.m3u

```diff
@@ -13,8 +13,8 @@ https://media.streambrothers.com:1936/8724/8724/playlist.m3u8
 https://gulfsat.cdn.easybroadcast.fr/live/Al-Sabah_abr/playlist.m3u8
 #EXTINF:-1 tvg-id="ATV.kw" status="online",ATV (360p) [Not 24/7]
 https://gulfsat.cdn.easybroadcast.fr/live/Aladalah_abr/playlist.m3u8
-#EXTINF:-1 tvg-id="Ch4Teen.kw" status="error",Ch4Teen (480p) [Not 24/7]
-https://dcunilive93-lh.akamaihd.net/i/dclive_1@835787/master.m3u8
+#EXTINF:-1 tvg-id="Ch4Teen.kw" status="online",Ch4Teen (480p) [Not 24/7]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/embed/video/x7tchkl
```

Comment:
Another channel link is also http://ch14.tv/live/ch4teenlive1/playlist.m3u8 

From list : 
http://ch14.tv/live/ch4teenclip/playlist.m3u8
http://ch14.tv/live/ch4teenquran/playlist.m3u8
http://ch14.tv/live/ch4teenkids/playlist.m3u8
http://ch14.tv/live/ch4teen1/playlist.m3u8
http://ch14.tv/live/ch4teenlive3/playlist.m3u8
http://ch14.tv/live/ch4teenlive2/playlist.m3u8
http://ch14.tv/live/ch4teenlive1/playlist.m3u8
http://ch14.tv/live/ch4teendoc/playlist.m3u8